### PR TITLE
Introduce signing payments, move output to src/data

### PR DESCRIPTION
**Pull request by jrwashburn**
Still not happy with how to handle keys, but this should work if they were provided in env. (I haven't actually tried with real keys, trying to decide how to best handle that.)

### CODE REVIEW

**Comment by jrwashburn** [comment]
Note currently sign will throw an exception which is simply caught and output to console.log.

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ dist
 node_modules
 .vscode
 .env
+./src/data/
```
Comment (bennoll) [inline .gitignore:7]: I don't think this works correctly with the leading dot.

File: src/core/sign.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import CodaSDK, { signPayment, keypair, payment } from "@o1labs/client-sdk";
+import { PayoutTransaction } from "./payouts";
+
+export async function signTransactionsToSend(payoutsToSign: PayoutTransaction[], keys: keypair): Promise<void> {
+    const payorPublicKey: string = process.env.PAYOR_PUBLIC_KEY || "";
+    let nonce = Number(process.env.STARTING_NONCE) || 0;
+    let fs = require('fs');
```
Comment (bennoll) [inline src/core/sign.ts:7]: Change to an `import`

File: src/index.ts [hunk 3]
```diff
@@ -19,6 +21,11 @@ async function main() {
   const slotsPerEpoch = Number(process.env.SLOTS_PER_EPOCH);
   const commissionRate = Number(process.env.COMMISSION_RATE);
   const transactionFee = Number(process.env.SEND_TRANSACTION_FEE) || 0;
+  const senderKeys: keypair = {
+    privateKey: process.env.PRIVATE_KEY || "",
+    publicKey: process.env.PUBLIC_KEY || ""
+  };
+
 
   var fs = require('fs');
```
Comment (bennoll) [inline src/index.ts:30]: Change to an `import`

**Review by bennoll — APPROVED** [review]

**Pull request merged**